app/assets/stylesheets/welcome.scss

```diff
@@ -2,32 +2,52 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
-.overlay-container {
-  position: relative;
-}
-
-.overlay-container .overlay {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  padding: 0.4rem;
-  // background-color: rgba(255, 255, 255, 0.7);
-  text-align: center;
-  font: 0/0 a;
-}
-
-.overlay-container .overlay:before {
-  content: ' ';
-  display: inline-block;
-  vertical-align: middle;
-  height: 100%;
-}
+.hero {
+  margin-bottom: 25px;
+  height: 80vh;
+  .intro {
+    padding: 3rem;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+  h1 {
+    color: #fff;
+    font-size: 1.5rem;
+    line-height: 1.5em;
+    letter-spacing: -0.025em;
+    font-weight: 300;
+    text-align: center;
+  }
+  p {
+    color: #fff;
+    line-height: 1.75em;
+    font-weight: 200;
+    text-align: center;
+    margin-bottom: 2rem;
+  }
+  .tech-img {
+    background: url('brick_interior.jpg') no-repeat center center;
+    background-size: cover;
+    height: 9rem;
```

Comment:
Properties should be ordered background, background-size, border-radius, display, height, margin, width